### PR TITLE
Object storage: Add metadata to indicate if the compiled crcmod was i…

### PR DESCRIPTION
**Pull request by stfeng**
…nstalled by PKB or not.(issue: 457) Fix azure package installation. (issue: 448)

### CODE REVIEW

**Comment by stfeng** [comment]
@yuyantingzero Please take a look. Fixing a couple of issues with object storage benchmark.

File: perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py [hunk 1]
```diff
@@ -775,6 +773,20 @@ def Prepare(self, vm):
                                                      DEFAULT_GCS_REGION,
                                                      self.regional_bucket_name))
 
+    # Detect if we need to install crcmod for gcp.
+    # See "gsutil help crc" for details.
+    raw_result, _ = vm.RemoteCommand('%s version -l' % vm.gsutil_path)
+    logging.info('gsutil version -l raw result is %s', raw_result)
+    search_string = 'compiled crcmod: True'
+    result_string = re.findall(search_string, raw_result)
+    if len(result_string) == 0:
```
Comment (voellm) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: This really should move to the packages directory as the crcmod package.

File: perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py [hunk 2]
```diff
@@ -775,6 +773,20 @@ def Prepare(self, vm):
...
+      logging.info('compiled crcmod is not available, installing now...')
+      vm.RemoteCommand('sudo pip install -U crcmod')
```
Comment (voellm) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: Should you ask for a specific version and/or collect the version info?

Comment (yuyantingzero) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: From documentation, uninstall crcmod first, then reinstall crcmod

Comment (stfeng) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: Done. Added the logic to uninstall first and then reinstall per documentation.

As to specific version, at this point I believe we should always get the latest version - I'm not worried about using the newest version of the crcmod, it's very stable (much more stable than Azure's python package which just reached 1.0). If it has a breaking change it'd break all gsutil users...

File: perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py (hunk 1 above)
Comment (stfeng) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: Done. I've made a package called crcmod under the packages directory and am calling vm.install(crcmod) here.

**Comment by yuyantingzero** [comment]
LGTM

**Pull request merged**